docker-compose.yml

```diff
@@ -34,26 +33,18 @@ services:
       PGPASSWORD: store_pgpass
       PGDATABASE: dev_database
       KANVAS_API_PORT: 3000
+      LOCAL_CORS: true
     env_file:
       - store-api-server/.env
     entrypoint: |-
       bash -c '
         set -e
-
         if [ "${STORE_API_ENABLED:-yes}" == "no" ]; then
           exit 0
         fi
-        INIT_QUEPASA=false INIT_PEPPERMINT=false ./script/migrate up
-
-        if [[ "`psql -c \"select count(1) from kanvas_user\" -tA`" == "0" ]]; then
-          if [ "${DEPLOYMENT:-no}" == "yes" ]; then
-            psql < script/populate-stagingdb.sql
-          else
-            psql < script/populate-testdb.sql
-          fi
-        fi
 
-        yarn start:prod
+        ./script/wait-db
```

Comment:
We don't need this `wait-db` logic anymore because we decided to rely on backoff retry.

Reply:
Based on my tests, in docker-compose local run, we need it due to `migration` and `seed`. Otherwise both database won't be updated and it needs to restart `store-api` once

Reply:
Removed and replaced by restart solution